src/platform_impl/ios/uikit/text_field.rs

```diff
@@ -0,0 +1,56 @@
+use super::UIView;
+use icrate::Foundation::{NSObject, NSString};
+use objc2::mutability::IsMainThreadOnly;
+use objc2::rc::Id;
+use objc2::runtime::{NSObjectProtocol, ProtocolObject};
+use objc2::{extern_class, extern_methods, extern_protocol, mutability, ClassType, ProtocolType};
+
+extern_class!(
+    #[derive(Debug, PartialEq, Eq, Hash)]
+    pub(crate) struct UITextView;
+
+    unsafe impl ClassType for UITextView {
+        #[inherits(NSObject)]
+        type Super = UIView;
+        type Mutability = mutability::InteriorMutable;
+    }
+);
+extern_methods!(
+    unsafe impl UITextView {
+        #[method(text)]
+        pub fn text(&self) -> &NSString;
+
+        #[method(setText:)]
+        pub fn setText(&self, text: &NSString);
+
+        #[method_id(@__retain_semantics Other delegate)]
```

Comment:
`@__retain_semantics` is a compilation-time unstable optimization that's only intended to be used by `icrate`. It works fine without it.

```suggestion
        #[method_id(delegate)]
```